### ISSUE TITLE
AddInstances() is referenced in docs but not available to ServiceRegistry

### ISSUE THREAD

**Issue by rtorres-olo** [JasperFx/lamar#341]
Has this been removed in newer versions? Are docs stale? https://jasperfx.github.io/lamar/guide/ioc/registration/registry-dsl.html#add-many-registrations-with-for-addinstances

**Comment by rtorres-olo** [comment]
user error sorry

**Comment by rtorres-olo** [comment]
not user error, I cannot seem to find this function

**Comment by rtorres-olo** [comment]
Am I supposed to use StructureMap in conjunction with Lamar for something like this?

**Comment by jeremydmiller** [comment]
@rtorres-olo No, it's an accident. There's some old StructureMap code in there for testing, and the docs are reading that old code. That method does not exist in Lamar.
